ui/pages/home/home.component.js

```diff
@@ -223,15 +234,27 @@ export default class Home extends PureComponent {
     return null;
   }
 
-  componentDidUpdate(_prevProps, prevState) {
-    const { closeNotificationPopup, isNotification } = this.props;
+  async componentDidUpdate(_prevProps, prevState) {
+    const {
+      closeNotificationPopup,
+      isNotification,
+      shouldShowSeedPhraseReminder,
+    } = this.props;
     const { notificationClosing } = this.state;
 
     if (notificationClosing && !prevState.notificationClosing) {
       closeNotificationPopup();
     } else if (isNotification) {
       this.checkStatusAndNavigate();
     }
+
+    const showSRP = await shouldShowSeedPhraseReminder;
```

Comment:
async prop here looks weird, I would rather prefer to put this await in parent class and pass result here